index.html

```diff
@@ -1,11 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="refresh" content="0;url=https://github.com/seriouslysean/monster-hunter-now-events">
```

Comment:
With the removal of this redirect, there needs to be a link to the GitHub homepage on this page somewhere, maybe as a very slim, centered footer.

Reply:
I'm confused, where is this file? I'd edited this out... and yes, I've added a footer with links

Reply:
This file is pushed to the hosted version of this repo, served via GitHub pages. If you added a link it’s all good!